cmd/cli/main.go

```diff
@@ -91,6 +106,35 @@ var rollbackCmd = &cobra.Command{
 	$ auditctl rollback -s 6dd1f926c346f06fc2c57d356ed648a2b518e74c`,
 }
 
+func getURL() string {
+	flags := []string{getAuthor, getSince, getUntil, getResource, getNamespace, getName}
+	flagnames := []string{"author", "since", "until", "resource", "namespace", "name"}
+	params := url.Values{}
+	for idx, flag := range flags {
+		params.Set(flagnames[idx], flag)
+	}
+	url := "http://" + serverAddr + "/changes"
+	reqUrl := fmt.Sprintf("%s?%s", url, params.Encode())
```

Comment:
one liner: `reqURL := fmt.Sprintf("http://%s/changes?%s", serverAddr, params.Encode())`

BTW, we avoid `reqUrl` as a variable name by convention, and we prefer `reqURL`. See https://github.com/golang/go/wiki/CodeReviewComments#initialisms